docs/landing_generated/index.html

```diff
@@ -7,6 +7,8 @@
     <title>Envoy: C++ L7 proxy and communication bus</title>
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic|Source+Code+Pro">
     <link href="stylesheets/site-00db0e0a.css" rel="stylesheet" />
+    <link rel="shortcut icon" type="image/png" href="/favicon.ico"/>
```

Comment:
Ah yeah will update that now as well. We should probably keep both around and maybe automate generating the landing_source as part of the build.